lib/gettext-setup/pot.rb

```diff
@@ -71,12 +71,11 @@ def self.generate_new_pot(locales_path = GettextSetup.locales_path, path = nil)
 
     def self.generate_new_po(language, locales_path = GettextSetup.locales_path,
                              pot_file = nil, po_file = nil)
+      language ||= ENV['LANGUAGE']
```

Comment:
Wait, hm? I was just saying move it down one line. Like swap it and the `initialize` line. So we initialize first, then check for all three env vars. Otherwise reading this code I would assume we needed language set before we initialized, which isn't true.